Assets/Scripts/UnityServices/Lobbies/LobbyAPIInterface.cs

```diff
@@ -42,6 +42,7 @@ public async Task<Lobby> CreateLobby(string requesterUasId, string lobbyName, in
             CreateLobbyOptions createOptions = new CreateLobbyOptions
             {
                 IsPrivate = isPrivate,
+                IsLocked = true, // locking the lobby at creation to prevent other players from joining before it is ready

```

Comment:
Just to confirm, the service disables this itself when the lobby allocation is ready?

Reply:
Not the service itself, but we do disable this. In LobbyServiceFacade.UpdateLobbyDataAsync() we lock/unlock the lobby depending on if the relay join code is available. In the past this was also how the lobby was locked after creation (because the Lobby SDK didn't include the possibility to lock at creation time).  Although now that we are only sending a single update when the host is ready, we can make this more explicit

Reply:
Ah yeah I couldn't find the reference, but there it is. Awesome.